scipy/special/boost_special_functions.h

```diff
@@ -721,12 +721,32 @@ template<typename Real>
 Real
 ncx2_cdf_wrap(const Real x, const Real k, const Real l)
```

Comment:
For long maintainability, it might be actually better to rename all these variables to x, df and nc throughout stats and special.